draft-ietf-trans-rfc6962-bis.md

```diff
@@ -1166,12 +1163,8 @@ Outputs:
     `inclusion_path` array of Merkle Tree nodes proves the inclusion of the
     chosen certificate in the selected STH.
 
-  sth:
-  : A base64 encoded `TransItem` of type `signed_tree_head_v2`, signed by this
-    log.
-
 > Note that no signature is required for the `inclusion` output as it is used to
-> verify inclusion in the selected STH, which is signed.
+> verify inclusion under the selected STH, which is signed.
```

Comment:
Let's be consistent.  "in" or "under", but not a mixture.  I see an occurrence of "inclusion...in the selected STH" just a few lines above.